apps/admin-portal/src/components/server-configurations/user-self-registration.tsx

```diff
@@ -23,7 +23,7 @@ import { EditSection, Hint, Section } from "@wso2is/react-components";
 import { useTranslation } from "react-i18next";
 import { Button, Container, Divider, Form, Modal } from "semantic-ui-react";
 import { Field, Forms, useTrigger } from "@wso2is/forms";
-import { getSelfSignUpConfigurations, updateSelfSignUpConfigurations } from "../../api/user-self-registration";
+import { getSelfSignUpConfigurations, updateSelfSignUpConfigurations } from "../../api/server-configurations";
```

Comment:
We can shorten this import if `server-configurations` is exported from the `api/index.ts`